upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -404,11 +404,27 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 				Version:           fi.String(version),
 				Selector:          networkingSelector,
 				Manifest:          fi.String(location),
-				KubernetesVersion: ">=1.6.0",
+				KubernetesVersion: "<1.7.0",
+				Id:                id,
+			})
+			manifests[key+"-"+id] = "addons/" + location
+		}
+
+		{
+			location := key + "/k8s-1.7.yaml"
```

Comment:
So we only _need_ a new manifest if the manifests are actually different.  1.6 is really >= 1.6, and pre-1.6 is really < 1.6.  The change being that 1.6 changed the node taints/tolerations (both the values and it moved from an annotation to a field), and that 1.6 introduced RBAC.

I'd prefer not to have a duplicate version if 1.7 & 1.6 are actually the same.  But of course, if they are different, then they have to be.

Are they different (and if so why :-) ) ?